lib/_emerge/AsynchronousLock.py

```diff
@@ -61,14 +51,11 @@ def _start(self):
                 return
 
         if self._force_process or (
-            not self._force_thread
-            and (self._use_process_by_default or threading is dummy_threading)
+            not self._force_thread and self._use_process_by_default
         ):
             self._imp = _LockProcess(path=self.path, scheduler=self.scheduler)
         else:
-            self._imp = _LockThread(
-                path=self.path, scheduler=self.scheduler, _force_dummy=self._force_dummy
-            )
+            self._imp = _LockThread(path=self.path, scheduler=self.scheduler)
```

Comment:
These two codepaths are now identical (both sides of the `if`/`else`).

I haven't checked, but it might be force_process or at least force_thread are obsolete now. (Verify!)

Reply:
One is `_LockProcess` and the other is `_LockThread`. I am not sure what differences they have. Maybe we should keep that change for a later time?